### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by alexdarie**
I think that this option is a better way to go since it is unobstrusive and is considered more standard.

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
It's arguable, yes, but the example still works.

You could also put the script element in the head and use the async attribute.

**Comment by alexdarie** [comment]
Oke, sure

File: popup-info-box-web-component/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Pop-up info box — web components</title>
+    <script src="main.js"></script>
```
Comment (brunoais) [inline popup-info-box-web-component/index.html]: I would add either `defer` or `async` to it

**Comment by Rumyra** [comment]
Hey @alexdarie ! Thank you for your pull request. We’re in the process of triaging pull requests across the mdn organization and due to the age of this one we’ll be closing it as out of date. If you think it’s still relevant, please open a new PR and check our [guidelines for contributing](CONTRIBUTING.md).

We appreciate your understanding.

**Pull request closed without merging**